cmd/process-agent/collector.go

```diff
@@ -197,6 +196,7 @@ func (l *Collector) runCheckWithRealTime(c checks.CheckWithRealTime, results, rt
 	}
 	l.messagesToResults(start, c.Name(), run.Standard, results)
 	if options.RunStandard {
+		runCounter := l.nextRunCounter(c.Name())
```

Comment:
nit: it may be a good idea to add a comment here to capture why we only update the counter during standard check runs.